### ISSUE TITLE
kubeflow namespace not found using kfctl.sh

### ISSUE THREAD

**Issue by IronPan** [kubeflow/kubeflow#1758]
I was following the instruction here 
https://www.kubeflow.org/docs/started/getting-started/
and last command failed 

```
${KUBEFLOW_REPO}/scripts/kfctl.sh apply k8s
ERROR handle object: can't update services kubeflow.ambassador: namespaces "kubeflow" not found
```
manually create namespace resolved the issue
kubeflow version 0.3.0

**Comment by tlkh** [comment]
Duplicate of [issues #1675 ](https://github.com/kubeflow/kubeflow/issues/1675)

Possibly addressed by [pull #1747 ](https://github.com/kubeflow/kubeflow/pull/1747)

**Comment by jlewi** [comment]
Fixed on the v0.3-branch. #1761 tracking bug for v0.3.1
